docker/Dockerfile.nvidia

```diff
@@ -129,59 +88,17 @@ RUN if [ "x$MPI_VER" = "x4" ]; then \
                $HPCSDK_HOME/comm_libs/mpi ; \
     fi;
 
-# Install pip dependencies and devito as a pip package
-RUN python3 -m venv /venv && \
-    /venv/bin/pip install --no-cache-dir --upgrade pip && \
-    /venv/bin/pip install --no-cache-dir wheel && \
-    /venv/bin/pip install --no-cache-dir -e /app/devito[extras,nvidia] && \
-    # Need extra CFLAGS for mpi4py
-    CFLAGS=-noswitcherror /venv/bin/pip install --no-cache-dir -r /app/devito/requirements-mpi.txt && \
-    rm -rf ~/.cache/pip
-
-ADD docker/run-jupyterlab.sh /jupyter
-ADD docker/run-tests.sh /tests
-ADD docker/run-print-defaults.sh /print-defaults
-ADD docker/entrypoint.sh /docker-entrypoint.sh
-ADD docker/nvdashboard.json /app/nvdashboard.json
+# Make devito env vars file
+RUN echo '#!/usr/bin/env bash' > /app/setup-devito-env.sh && \
```

Comment:
We have an `Dockerfile.nvidia.clang` because it installs additional (llvm) things. A dockerfile just for env vars is overkill. Once we have CUDA we would add an `ARG` for language but that's about it.